src/test/java/uk/gov/hmcts/reform/roleassignment/helper/TestDataBuilder.java

```diff
@@ -484,7 +484,22 @@ public static ExistingRoleAssignment buildExistingRoleForIAC(String actorId, Str
             .grantType(GrantType.STANDARD)
             .attributes(attributes)
             .build();
+    }
 
+    public static ExistingRoleAssignment buildExistingRoleForSSCS(String actorId, String roleName,
```

Comment:
Can we please reuse or extend the existing methods instead of creating new one for SSCS?

Reply:
ok ill look into it